### DISCUSSION TITLE
Editing OHIF Directly

### DISCUSSION THREAD

**Discussion by joeadham** [Project-MONAI/MONAILabel#1668] [Q&A] [answered]
Hello, I was wondering if it's possible to edit the OHIF files directly in the monailabel package. I've attempted it before, but I couldn't locate the right JavaScript files that match those in the current stable build by OHIF.

For example, I would like to enable the upload DICOM OHIF functionality, where you could usually enable it in the: `platform/app/public/config/docker_nginx-orthanc.js` file. by adding the following lines:

```

customizationService: {
    dicomUploadComponent:
      '@ohif/extension-cornerstone.customizationModule.cornerstoneDicomUploadComponent',
  },
dicomUploadEnabled: true,

```
But I couldn't find the `platform/app/public/config/docker_nginx-orthanc.js` file anywhere in the package.

**Comment by diazandr3s** [comment, marked as answer]
Hi @joeadham,

Good question!

Actually, you could use OHIF independently of MONAI Label. In this repo, we host the [extension and mode of MONAI Label for OHIF](https://github.com/Project-MONAI/MONAILabel/tree/main/plugins/ohifv3). In this script we install the OHIF before linking both the extension and mode: https://github.com/Project-MONAI/MONAILabel/blob/main/plugins/ohifv3/build.sh#L24-L30

You could start the latest OHIFv3 and then link the MONAI Label extension and mode: https://github.com/Project-MONAI/MONAILabel/tree/main/plugins/ohifv3

Here is more information about the OHIF commands: https://docs.ohif.org/development/ohif-cli/

Hope this helps,

**Reply by joeadham** [reply]
Thank you, that's exactly what I needed.
